deployment/templates/secret.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.development }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: lodestar-artifacts-configuration
+type: Opaque
+stringData:
+  GITLAB_API_URL: "{{ .Values.api.gitlab }}"
```

Comment:
I think the ENGAGEMENT_API_URL is missing from here and the values-dev.yaml. If you want to merge this I can handle the last bits of this initial cut.

Reply:
added engagement api to secret and values-dev.yaml